project/MiMa.scala

```diff
@@ -0,0 +1,94 @@
+// It would be nice to use sbt-mima-plugin here, but the plugin is missing
+// at least two features we need:
+// * ability to run MiMa twice, swapping `curr` and `prev`, to detect
+//   both forwards and backwards incompatibilities
+// * ability to pass a filter file
```

Comment:
I think this is tracked in https://github.com/typesafehub/migration-manager/issues/102. Perhaps it would be useful to add that as a link in the comment.